### PR TITLE
ADST-422 OIDC. changes for Auth0

### PR DESCRIPTION
**Pull request by aman-alfresco**
- added auth0 authentication

### CODE REVIEW

**Review by PiplaniVinay — CHANGES_REQUESTED** [review]
Please change the branch from main to OIDC ticket, also convert the ACS-7518 jira to ADST.

**Comment by aman-alfresco** [comment]
> Please change the branch from main to OIDC ticket, also convert the ACS-7518 jira to ADST.

File: settings.gradle
[the remark this answers is not in the document]
Comment (aman-alfresco) [reply settings.gradle:1]: Done

File: session/src/main/kotlin/com/alfresco/content/session/Session.kt
[the remark this answers is not in the document]
Comment (aman-alfresco) [reply session/src/main/kotlin/com/alfresco/content/session/Session.kt:1]: Done

**Review by gl-lovekesh — APPROVED** [review]

**Pull request merged**